src/main/java/com/monumental/services/MonumentRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.monumental.services;
+
+import com.monumental.models.Monument;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+
+@Repository
+@Transactional
+public interface MonumentRepository extends CrudRepository<Monument, Integer> {
+
+}
```

Comment:
Is this being used anywhere right now? Or will that come in your next PR?

Reply:
Not being used. We will definitely have one for all the models eventually